### PR TITLE
All my codes so far

### PR DESCRIPTION
**Pull request by sjsakib**
Some problems are solved with python.

### CODE REVIEW

File: STL/UVa 10815 - Andy's First Dictionary/UVa10815.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+"""
```
Comment (shanto86) [inline STL/UVa 10815 - Andy's First Dictionary/UVa10815.py]: Python file does not take space? Why separate folder?

File: STL/UVa 10815 - Andy's First Dictionary/UVa10815.cpp [hunk 2]
```diff
@@ -0,0 +1,35 @@
+/**
```
Comment (shanto86) [inline STL/UVa 10815 - Andy's First Dictionary/UVa10815.cpp]: Why separated folder?

File: EasyProblems/Timus 1005 - Stone Pile/coin-change-dp.cpp [hunk 3]
```diff
@@ -0,0 +1,35 @@
+/**
```
Comment (shanto86) [inline EasyProblems/Timus 1005 - Stone Pile/coin-change-dp.cpp]: please don't make separated folder. Do something like v1 v2 in the name, and put a comment on top of the file to indicate solving method.

File: EasyProblems/Exercises/UVa815.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
+"""
```
Comment (shanto86) [inline EasyProblems/Exercises/UVa815.py]: Having separate folder for Exercises makes sense, but is it compatible with the existing ones/others? If not can you please send another change list to make it uniform?

**Comment by shanto86** [comment]
Thank you for your solutions. Can you please-
1. Add problem name to the file name? In case you make mistake in problem number, it is highly unlikely that you will also mistake in problem name.
2. Make sure the folder/file formatting of the existing ones are same? Otherwise can you make them same?

File: EasyProblems/Timus 1005 - Stone Pile/coin-change-dp.cpp (hunk 3 above)
Comment (sjsakib) [reply EasyProblems/Timus 1005 - Stone Pile/coin-change-dp.cpp]: ok, I thought of keeping all the codes for one problem in one place

**Comment by sjsakib** [comment]
OK

**Comment by sjsakib** [comment]
Thanks! :)

**Pull request merged**